target/linux/ath79/image/generic.mk

```diff
@@ -75,6 +75,16 @@ define Device/seama
   SEAMA_SIGNATURE :=
 endef
 
+define Device/airtight_c-65
+  ATH_SOC := qca9558
+  DEVICE_TITLE := Airtight C-65
```

Comment:
Please update to use DEVICE_VENDOR, DEVICE_MODEL and DEVICE_VARIANT as introduced in:
https://github.com/openwrt/openwrt/commit/8c3dbaf8b49233e275b2dce4e30dc0a2bf8c86c5

Reply:
Done